front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -1,7 +1,6 @@
 import React, { useState, useEffect } from 'react';
 import { convertUrlToUploadFile } from 'utils/fileUtils';
 import { useParams } from 'react-router-dom';
-// components
```

Comment:
Why do you remove it here, but still adding in other places? https://github.com/CitizenLabDotCo/citizenlab/pull/2569/files#diff-ae98b6522d02a467e529bf0b6aec86e2ab74f3fa394ba61f1d0350a505933796R13

Reply:
It doesn't matter. Imo it doesn't add much (if any) value imo. I always use search + linter errors anyway to find stuff (and everyone probably should)

Reply:
Ok for me :slightly_smiling_face: BTW there was a discussion about it recently https://citizenlabco.slack.com/archives/CQAGMHKT3/p1660302549511009

Reply:
Lol, added it back if it gives people peace of mind.